pmd-java/src/main/resources/rulesets/java/metrics.xml

```diff
@@ -16,51 +16,46 @@
           metrics="true"
           externalInfoUrl="${pmd.website.baseurl}/rules/java/codesize.html#CyclomaticComplexity">
         <description>
-            <![CDATA[
-Complexity directly affects maintenance costs is determined by the number of decision points in a method
-plus one for the method entry.  The decision points include 'if', 'while', 'for', and 'case labels' calls.
-Generally, numbers ranging from 1-4 denote low complexity, 5-7 denote moderate complexity, 8-10 denote
-high complexity, and 11+ is very high complexity.
-    ]]>
+            The complexity of methods directly affects maintenance costs and readability. Concentrating too much decisional logic
```

Comment:
We should keep the "<![CDATA[" around - this preserves the whitespace, which keeps the paragraphs separated, when the rule documentation is generated.